source/_data/themes.yml

```diff
@@ -1,3 +1,17 @@
+- name: leedom 
+  description: A pure hexo theme. 
+  link: https://github.com/leedom92/hexo-theme-leedom 
+  preview: https://blog.leedom.me/ 
+  tags: 
+  - 中文
+  - Chinese
+  - English
+  - Pure
+  - Simple
+  - Concise
+  - Search
+  - Evan You
+  - 尤雨溪
```

Comment:
Do you really, really have permission to use @yyx990803's real name here?

We can't merge your pull request if you can not provide proof that you do have his permission.

Reply:
@SukkaW Hi, I have resolved the problem, plz merge my pr, thx!